### ISSUE TITLE
How to use

### ISSUE THREAD

**Issue by ramiel** [lab360-ch/verdaccio-gitlab-ci#1]
Hello, can I have some explanation on how to use this plugin?

**Comment by juanpicado** [comment]
Agree, I think readme should be a bit more self-explanatory. Some sort of example and the main objective of this plugin (even if the name seems to be self-explanatory).

**Comment by OKNoah** [comment]
@juanpicado Do you know how it works?
